hatchet.json

```diff
@@ -7,5 +7,6 @@
     "kissaten/java-servlets-sample",
     "kissaten/webapp-runner-sample",
     "kissaten/korvan"
-  ]
+  ],
+  "github": ["heroku/java-getting-started"]
```

Comment:
This can go now, right?

Reply:
Yes, but one of the tests is still failing because we need

Reply:
Yes, but one of the tests is still failing because we need an app with tests that the jvm-common buildpack can execute. Is there already one in the existing suite that executes CI?